src/webgpu/api/validation/render_pass/resolve.spec.ts

```diff
@@ -46,6 +47,20 @@ Test various validation behaviors when a resolveTarget is provided.
       resolveTargetWidth: 4,
       _valid: true,
     },
+    // zero resolve target mip level count should be invalid with generating a validation error when
+    // creating a texture view.
+    {
+      resolveTargetViewBaseMipLevel: 1,
+      resolveTargetViewMipCount: 0,
+      _valid: false,
+    },
+    // zero resolve target array layer count should be invalid with generating a validation error
+    // when creating a texture view.
+    {
+      resolveTargetViewArrayLayerCount: 0,
+      resolveTargetViewBaseArrayLayer: 1,
+      _valid: false,
+    },
```

Comment:
Instead of these 2 new cases how about just one new case: `{ resolveTargetInvalid: true, _valid: false }`

Reply:
Done.